packages/monitoring/src/serviceMonitor/tfChain.ts

```diff
@@ -0,0 +1,39 @@
+import { KeypairType } from "@polkadot/util-crypto/types";
+import { Client as TFClient } from "@threefold/tfchain_client";
+
+import { IDisconnectHandler, ILivenessChecker, ServiceStatus } from "../types";
+
+export class TFChainMonitor implements ILivenessChecker, IDisconnectHandler {
+  private name = "TFChain";
+  private url: string;
+  private tfClient: TFClient;
+  constructor(tfChainUrl: string, mnemonic: string, keypairType: KeypairType) {
+    this.url = tfChainUrl;
+    this.tfClient = new TFClient({ url: this.url, mnemonicOrSecret: mnemonic, keypairType: keypairType });
```

Comment:
why are we using TFClient here? it requires a mnemonic and a keypair type. Instead, we can use the queryClient which requires only the url